salt/utils/sanitisers.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
```

Comment:
I tend to defer style/standards decisions to @thatch45. I don't really care which way we go, I'm more concerned with consistency.